src/Compatibility/Core/src/Windows/Platform.cs

```diff
@@ -136,10 +137,10 @@ internal Platform(Microsoft.UI.Xaml.Window page)
 
 			_container.SizeChanged += OnRendererSizeChanged;
 
-			MessagingCenter.Subscribe(this, Page.BusySetSignalName, (Page sender, bool enabled) =>
+			WeakReferenceMessenger.Default.Register<UI.Xaml.Window, PageBusyMessage>(page, (window, message) =>
```

Comment:
The recipient is not used, we can drop the first type argument:

```suggestion
			WeakReferenceMessenger.Default.Register<PageBusyMessage>(page, (_, message) =>
```